src/main/java/com/laytonsmith/core/AliasCore.java

```diff
@@ -678,8 +757,7 @@ public static String file_get_contents(String fileLocation) throws IOException {
 	public static boolean file_put_contents(File fileLocation, String contents, String mode)
 			throws Exception {
 		BufferedWriter out = null;
```

Comment:
The `null` value here is not used, and you've correctly removed the handling of the never-happening `null` value. You should however also remove the `null` assign here, as it is never used.